day2/day2.js

```diff
@@ -0,0 +1,54 @@
+// (() => { anon arrow function
+// })();
+//var { input } = require(`./input.js`);
+
+const convertInput = (input) => {
+  let command = input.split('\n ');
+  arr = [];
+  command.forEach((element) => {
+    let output = element.split(' ');
+    let obj = {};
+
+    obj.direction = output[0];
+    obj.distance = parseInt(output[1]);
+    arr.push(obj);
```

Comment:
I'd suggest to add some error handling in here so if there are lines which don't follow the expected pattern it wouldn't crash.

Imagine there are lines like these and try to handle them:

- forward a
- 3
- down
- up4
- forward 3up 4